compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -127,37 +130,57 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
     break;
   case IREE::Codegen::DispatchLoweringPassPipeline::
       CPUBufferOpsTileAndVectorize: {
-    TilingConfig tilingConfig = getTilingConfigForPipeline(funcOp);
-    addCPUBufferOpsTileAndVectorizePipeline(pipeline, tilingConfig,
+    FailureOr<TilingConfig> tilingConfig = getTilingConfigForPipeline(funcOp);
+    if (failed(tilingConfig)) {
+      return signalPassFailure();
+    }
```

Comment:
This code is duplicated for all cases except default and none. Maybe split into two `switch` statements?